pynicotine/gtkgui/ui/userbrowse.ui

```diff
@@ -268,7 +267,7 @@
                       <object class="GtkBox">
                         <property name="visible">1</property>
                         <child>
-                          <object class="GtkScrolledWindow">
+                          <object class="GtkScrolledWindow" id="FileScrolledWindow">
```

Comment:
This id tag is needed for the `FileScrolledWindow.get_hadjustment()` function in userbrowse.py which enables the use of the "Left" arrow key for both widget focusing and horizontal scrolling.